### PR TITLE
Update node 8.8.0 pre

### PR DESCRIPTION
**Pull request by kderme**
# Description

Add your description here, if it fixes a particular issue please provide a [link](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=) to the issue.

# Checklist

- [ ] Commit sequence broadly makes sense
- [ ] Commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] Any changes are noted in the [changelog](https://github.com/IntersectMBO/cardano-db-sync/blob/master/cardano-db-sync/CHANGELOG.md)
- [ ] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) on version 0.10.1.0 (which can be run with `scripts/fourmolize.sh`)
- [ ] Self-reviewed the diff

# Migrations

- [ ] The pr causes a [breaking change](https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/migrations.md) of type a,b or c
- [ ] If there is a breaking change, the pr includes a database migration and/or a fix process for old values, so that upgrade is possible
- [ ] Resyncing and running the migrations provided will result in the same database semantically

If there is a breaking change, especially a big one, please add a justification here. Please elaborate
more what the migration achieves, what it cannot achieve or why a migration is not possible.

### CODE REVIEW

**Comment by sgillespie** [comment]
We seem to have lost cardano-chain-gen jobs in Hydra: https://ci.iog.io/eval/32753?compare=master&full=0

**Comment by kderme** [comment]
> We seem to have lost cardano-chain-gen jobs in Hydra: https://ci.iog.io/eval/32753?compare=master&full=0

I accidentally removed them, will fix.

File: cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Alonzo.hs [hunk 1]
```diff
@@ -271,18 +285,18 @@ handleCertPtr rdmrIx dcert mps =
...
-  (Ledger.Era era) =>
+  DBPlutusScript era =>
   (ScriptHash StandardCrypto, Alonzo.AlonzoScript era) ->
   TxScript
 mkTxScript (hsh, script) =
   TxScript
     { txScriptHash = unScriptHash hsh
     , txScriptType = getScriptType
-    , txScriptPlutusSize = getPlutusScriptSize script
+    , txScriptPlutusSize = Nothing -- getPlutusScriptSize script TODO: Conway
```
Comment (Cmdv) [inline cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Alonzo.hs:299]: is this to be done in another PR?

File: cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Babbage.hs [hunk 2]
```diff
@@ -29,6 +29,7 @@ import qualified Cardano.Ledger.Core as Core
 import qualified Cardano.Ledger.Era as Ledger
 import Cardano.Ledger.Mary.Value (MaryValue (..), MultiAsset (..))
 import qualified Cardano.Ledger.Plutus.Data as Alonzo
+import qualified Cardano.Ledger.Plutus.Data as Plutus
```
Comment (Cmdv) [inline cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Babbage.hs:32]: seems a little confusing to qualify the same module as two different names:
```
import qualified Cardano.Ledger.Plutus.Data as Alonzo
import qualified Cardano.Ledger.Plutus.Data as Plutus
```

**Review by Cmdv — APPROVED** [review]
LGTM just a couple of questions

File: cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Alonzo.hs (hunk 1 above)
Comment (kderme) [reply cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx/Alonzo.hs]: Yes, done!

File: cardano-chain-gen/src/Cardano/Mock/Forging/Tx/Alonzo/ScriptsExamples.hs [hunk 3]
```diff
@@ -114,7 +129,7 @@ plutusDataEncLen = toShort $ toStrictByteString $ mconcat (encodeListLen 2 : (en
 plutusDataEncIndef :: ShortByteString
 plutusDataEncIndef = toShort $ toStrictByteString $ encodeList plutusData2
 
-toBinaryPlutus :: AlonzoScript era -> BinaryPlutus
-toBinaryPlutus as = case as of
-  TimelockScript _ -> panic "expected Alonzo script"
-  PlutusScript (Plutus _ sbs) -> sbs
+-- toBinaryPlutus :: AlonzoEraScript era => AlonzoScript era -> PlutusBinary
```
Comment (sgillespie) [inline cardano-chain-gen/src/Cardano/Mock/Forging/Tx/Alonzo/ScriptsExamples.hs:132]: What's this?

File: cardano-db-sync/src/Cardano/DbSync/Fix/PlutusScripts.hs [hunk 4]
```diff
@@ -108,16 +110,17 @@ findWrongPlutusScripts tracer =
...
+      let script :: AlonzoScript StandardAlonzo = PlutusScript (AlonzoPlutusV1 (Plutus $ PlutusBinary $ SBS.toShort bytes))
       let hsh :: Ledger.ScriptHash StandardCrypto = Ledger.hashScript @StandardAlonzo script
       Right $ Generic.unScriptHash hsh
-      where
-        getLang = case DB_V_13_0.scriptType dbScript of
-          PlutusV1 -> Right Ledger.PlutusV1
-          PlutusV2 -> Right Ledger.PlutusV2
-          _ -> Left "Non plutus script found where it shouldn't."
+
+--      where
```
Comment (sgillespie) [inline cardano-db-sync/src/Cardano/DbSync/Fix/PlutusScripts.hs:119]: What's this?

**Comment by sgillespie** [comment]
I realize it's a bit too late, but I had some questions on the commented-out code. Otherwise, LGTM
